src/main/java/pers/gwyog/gtneioreplugin/util/DimensionHelper.java

```diff
@@ -94,15 +95,9 @@
             s = s.trim();
             for (int i = 0; i < DimNameDisplayed.length; i++) {
                 if (s.equals(DimNameDisplayed[i])) {
-                    s = DimName[i].replaceAll("GalacticraftCore_", "").replaceAll("GalacticraftMars_", "").replaceAll("GalaxySpace_", "").replaceAll("Vanilla_", "Vanilla ");
-                    if (s.equals("Twilight"))
-                        s = "Twilight Forrest";
-                    else if (s.equals("Underdark"))
-                        s = "Deep Dark";
-                    else if (s.equals("EndAsteroid"))
-                        s = "Far End Asteroids";
-                    
-                    switch (s) {
+                    s = I18n.format("gtnop.world." + DimName[i].replaceAll("GalacticraftCore_", "").replaceAll("GalacticraftMars_", "").replaceAll("GalaxySpace_", "").replaceAll("Vanilla_", "Vanilla "));
```

Comment:
Shouldn't this be StatCollector something? Not quite sure if it would make a difference, but StatCollector is the usual choice.

Reply:
Glease, StatCollector is commly used on server side (removed in newer mc versions), while I18n.format is the new mechanic for localizing stuff on client side.

Reply:
That'd be good I guess